doc/content/getting-started/migrating/migrating-from-v2/migrate-using-console.md

```diff
@@ -10,9 +10,11 @@ This section refers to migrating end devices to {{% tts %}} by using [{{% tts %}
 
 Migrating your end devices using {{% tts %}} Console is easy, but it is convenient only if you have an intention of migrating few devices. 
 
-{{< note >}} This method comes down to adding end devices in {{% tts %}} Console one by one, and it is not possible to migrate groups of devices as with using `ttn-lw-migrate` tool. 
+{{< note >}} This method comes down to adding end devices in {{% tts %}} Console one by one, and it is not possible to migrate groups of devices as with using `ttn-lw-migrate` tool. {{</ note >}}
```

Comment:
This is not correct. You can import multiple devices in the console. Let's remove this part as it only causes confusion.